tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "skipLibCheck": true,
     "resolveJsonModule": true,
     "jsx": "react",
-    "lib": ["es6", "dom"],
+    "lib": ["es2019", "dom"],
```

Comment:
out of curiosity - what has prompted this bump?

Reply:
`.flatMap()` mostly, and also keeping up with the times ⌚